compiler/lib/js_output.ml

```diff
@@ -2012,6 +2013,7 @@ let program ?(accept_unnamed_var = false) f ?source_map p =
                   Source_map.Gen_Ori_Name
                     { gen_line; gen_col; ori_source; ori_line; ori_col; ori_name })
         in
+        let mappings = Source_map.Mappings.encode mappings in
```

Comment:
@OlivierNicole Isn't this going to be quadratic? If we have `n` files, we are going to decode and encode a larger and larger sourcemap `n` times.

Reply:
This is gone in #1715